src/widget/wtracktableview.cpp

```diff
@@ -565,6 +570,15 @@ void WTrackTableView::mouseMoveEvent(QMouseEvent* pEvent) {
     if (!trackModel) {
         return;
     }
+
+    const QPointF delta = m_mousePressPosition - pEvent->position();
+    const qreal threshold = 5;
+    const qreal distanceSinceMousePress = std::sqrt(delta.x() * delta.x() + delta.y() * delta.y());
```

Comment:
there's a shorthand [QPointF::manhattanLength()](https://doc.qt.io/qt-6/qpointf.html#manhattanLength)

Reply:
According to the documentation, the manhattan length is the sum of x and y. Here we need the hypotenuse.

Reply:
Oh, sorry for the noise :roll_eyes:, I was confused by that snippet in https://doc.qt.io/qt-6/qpoint.html#manhattanLength
> This is a useful, and quick to calculate, approximation to the true length:
> 
> > double trueLength = std::sqrt(std::pow(x(), 2) + std::pow(y(), 2));